src/containers/Landing/index.tsx

```diff
@@ -1,21 +1,24 @@
 import { styled } from '@mui/material/styles';
+import Notification from '~/components/Notification';
```

Comment:
it should be
```suggestion
import Notification from '~/components';
```

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/2/commits/ba0c3959874ad8a573436f5456e5d6253b4026d2